### DISCUSSION TITLE
Index.html?

### DISCUSSION THREAD

**Discussion by colo13verdadeo** [ImpressCMS/impresscms#922] [Q&A] [answered]
Hello team, i need run website to test segurity, and with index.html dont work.
what is the main page?

I didn't find solution on "https://github.com/ImpressCMS/impresscms/blob/master/README.md"

My english could be bad, srry.

Image: https://imgur.com/yu49jMo

**Comment by fiammybe** [comment]
Hi @colo13verdadeo , it depends on the version of ImpressCMS that you want to install. The installation guide for ImpressCMS 1.4.x is here : https://www.impresscms.org/modules/simplywiki/index.php?page=Installation_Guide , and the installation guide for ImpressCMS 2.X (the development version that is still in alpha) can be found here : https://www.impresscms.org/modules/simplywiki/index.php?page=Installing_ImpressCMS_2

**Comment by fiammybe** [comment, marked as answer]
In the meantime, I added a section the the readme concerning installations for the 2.0 alpha.

Normally, if your web server is configured correctly, it should automatically load the index.php in the main folder (which is in the htdocs subfolder for impresscms 2.0) and the installer should start. 

Let me know if you have any issues, and the webserver (make + version) and php version would be good to know in that case.
